urls_test.go

```diff
@@ -21,6 +21,78 @@ import (
 	"testing"
 )
 
+func TestIsURL(t *testing.T) {
```

Comment:
I don't really see the value of `TestIsURL`. As it currently stands, it is just change detector for the unexported parseURL function, which is itself unit tested.

I'm not opposed to keeping it though.

Reply:
Agreed, all the same test cases as before.